src/SfxWeb/src/app/views/node/essentials/essentials.component.ts

```diff
@@ -98,15 +88,25 @@ export class EssentialsComponent extends NodeBaseControllerDirective {
       this.node.loadInformation.refresh(messageHandler),
       this.node.deployedApps.refresh(messageHandler).pipe(map(() => {
         this.deployedApps = this.node.deployedApps;
-      })),
-      this.data.clusterManifest.ensureInitialized().pipe(mergeMap(() => {
-        if (this.data.clusterManifest.isRepairManagerEnabled) {
-          return this.data.repairCollection.refresh().pipe(map(() => {
-            this.repairJobs = this.data.repairCollection.getRepairJobsForANode(this.node.name);
-          }));
-        }else {
-          return of(null);
+        let warning = 0;
+        let ok = 0;
+        let error = 0;
+        for(let i = 0; i <  this.deployedApps.data.apps.collection.length; i++) {
+          this.healthStatus = this.deployedApps.data.apps.collection[i].data.apps.healthState;
+          if(this.healthStatus.text == 'Error') {
```

Comment:
So I can create the object but the if-statement is going to be needed because the issue with using this `${data.apps.healthState.text}Count` is that the text for healthState that is returned is "OK" and not "Ok" So the best way it seems to do this is `if(healthStatus.text == 'Error') {
            healthCount["ErrorCount"]++;
          } else if(healthStatus.text == 'Warning') {
            healthCount["WarningCount"]++;
          } else if(healthStatus.text == 'OK') {
            healthCount["OkCount"]++;
          }
        }
        this.deployedDashboard = DashboardViewModel.fromHealthStateCount("Deployed", "Deployed", false, healthCount);
      }))`